stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
```

Comment:
@meh - Do you think $$table must be a JS object literal or can it be a Ruby Hash?